### PR TITLE
chore: move all NullxxxColumn functions to `dbutil` package

### PR DESCRIPTION
**Pull request by sashaostrikov**
Follow up for https://github.com/sourcegraph/sourcegraph/pull/42818

## Test plan
Existing tests.

### CODE REVIEW

**Review by unknwon — APPROVED** [review]

File: internal/database/dbutil/dbutil.go [hunk 1]
```diff
@@ -76,6 +83,13 @@ func (nt NullString) Value() (driver.Value, error) {
 	return *nt.S, nil
 }
 
+func NullStringColumn(s string) *string {
```
Comment (mrnugget) [inline internal/database/dbutil/dbutil.go:86]:
```suggestion
// NullStringColumn represents a string that should be inserted/updated as NULL when blank.
func NullStringColumn(s string) *string {
```

File: internal/database/dbutil/dbutil.go [hunk 2]
```diff
@@ -44,6 +44,13 @@ func (nt NullTime) Value() (driver.Value, error) {
 	return *nt.Time, nil
 }
 
+func NullTimeColumn(t time.Time) *time.Time {
```
Comment (mrnugget) [inline internal/database/dbutil/dbutil.go:47]:
```suggestion
// NullTimeColumn represents a timestamp that should be inserted/updated as NULL when t.IsZero() is true.
func NullTimeColumn(t time.Time) *time.Time {
```

File: internal/database/dbutil/dbutil.go [hunk 3]
```diff
@@ -104,6 +118,13 @@ func (n NullInt32) Value() (driver.Value, error) {
 	return *n.N, nil
 }
 
+func NullInt32Column(n int32) *int32 {
```
Comment (mrnugget) [inline internal/database/dbutil/dbutil.go:121]:
```suggestion
// NullInt32Column represents an int32 that should be inserted/updated as NULL when the value is 0.
func NullInt32Column(n int32) *int32 {
```

File: internal/database/dbutil/dbutil.go [hunk 4]
```diff
@@ -140,6 +161,13 @@ func (n NullInt64) Value() (driver.Value, error) {
 	return *n.N, nil
 }
 
+func NullInt64Column(n int64) *int64 {
```
Comment (mrnugget) [inline internal/database/dbutil/dbutil.go:164]:
```suggestion
// NullInt64Column represents an int64 that should be inserted/updated as NULL when the value is 0.
func NullInt64Column(n int64) *int64 {
```

**Review by mrnugget — APPROVED** [review]
Thanks for doing that!!

**Pull request merged**